src/common/options/rbd-mirror.yaml.in

```
@@ -2,6 +2,13 @@
 ---
 
 options:
+- name: rbd_mirror_remote_osd_op_timeout
+  type: uint
+  level: advanced
+  desc: timeout for remote operations handled by osds such as write (0 is unlimited)
```

I don't think we want to emphasize writes when 99% percent of remote OSD requests are read-like (actual reads, list-snaps, etc):
```suggestion
  desc: timeout for OSD operations against the remote cluster (0 is unlimited)
```